en/protocol/parameter.md

```diff
@@ -1,7 +1,64 @@
 # Parameter Protocol
 
+> **Caution** This content has not been reviewed since being ported from the old website (and may be out of date). Updates/re-validation welcome!
+
 The parameter protocol is used to exchange key system settings and guarantees delivery.
 
+The onboard parameter interface allows to read and write parameters (e.g. PID gains) into the current memory (RAM) and also into permanent storage.
+
+It can be both implemented on a microcontroller (e.g. the pxIMU with ARM7) and in standard software (e.g. px_multitracker process in Linux).
+
+## Supported Data Types
+
+MAVLink v1.0 supports these data types:
```

Comment:
Are there any new data types for MAVlink v2.0? If so, what are they? IF not, should we just make this sentence MAVLink (v1.0, v2.0) supports these data types:

Reply:
I think that applies for both, correct.

Reply:
Thanks. Done.